### ISSUE TITLE
Accessibility Issues with YouTube iframe

### ISSUE THREAD

**Issue by terrill** [uweb/uw-2014#35]
When adding a YouTube video (which presumably uses a YouTube plugin or widget), the embed code provided by YouTube has some accessibility errors. Until Google addresses this problem, could these be fixed dynamically after the widget is loaded? The errors: 
- The `<iframe>` element includes frameborder="0", which is obsolete and fails HTML5 validation. It also has width and height attributes, which are valid but could also be replaced with CSS, thereby attaining full separation of content from presentation. 
- The `<iframe>` element needs a title attribute. I recommend title="YouTube video player"

**Comment by lcaple** [comment]
Addressed in #95. Closing.
